### ISSUE TITLE
Feature: Offer four original dubs

### ISSUE THREAD

**Issue by Richard-L** [LostArtefacts/TRX#1021]
Related: #1610
Related: #2039
Related: #4618

One of the few things the console versions still hold over TR1X, and since we've talked about it a couple of times over the past 2 years but have no issue yet, I thought I'd create one.

Released were:
- Japanese
- English
- German
- French

^ Where the PS1 discs will be the best quality sources, as they contain all speech as 44.1khz audio tracks. I could probably make them available. The audio streams inside the FMV containers similarly have the best quality on the PS1 releases.

I had discovered some curiosities with the GYM level, where the Japanese dub goes into quite some detail by addressing the specific controller symbols on the PS1 and Saturn controllers respectively. This could be a rather niche but thus nice touch to include perhaps. Similarly, the PC versions address some keys differently, albeit audio quality from the different `GYM.PHD`s is too low (11.205khz). For those interested, I'd posted samples and more details here: https://discord.com/channels/1071085441077481544/1075372522377322516/1157653786999930991

Example of how OpenLara includes this feature (and combines it with Subtitles https://github.com/LostArtefacts/TR1X/issues/382 ):

https://github.com/LostArtefacts/TR1X/assets/4934209/be70f289-8f34-48a2-903f-007f24a1b426

**Comment by aredfan** [comment]
Great to see an issue for this.  I thought it might be worth adding a couple of relevant/useful links.

-Spreadsheet for all known TR1 tracks:
https://docs.google.com/spreadsheets/d/1xQsqGF-T0oqv_TxdgAMJr31XDiJrnch5tuas6QpXzsI/edit#gid=505009050

-Localization for TombATI:
https://www.tombraiderforums.com/showthread.php?t=213154

**Comment by VGMFR** [comment]
Actually this can probably be split between 3 différents subjects / features:
- Global localization, which is probably the easiest of the 3. TombATI handles it in a quite neat way. This could maybe be a new issue. Edit: there is already an issue for that: https://github.com/LostArtefacts/TR1X/issues/715
- Localization for data tracks. Actually it is already possible to have you tracks localized, if you download them from the ATI thread and then replace your data with the localized one. I currently have my cutscenes and dialogs in French and it was quite easy to do. But being able to easily switch in the menu or configuration and combine that with the global localization would be way better. This is definitively this issue.
- The subtitles, which is an other subject but somehow related as pointed beforehand in this thread (https://github.com/LostArtefacts/TR1X/issues/382)

**Comment by Kappa971** [comment]
@rr-, there's no need to add any support for localized audio in other languages ​​because it already works in TRX: you just need to have the FMVs and audio tracks with the non-English dubbing.
If you'd like, I can upload videos of Tomb Raider 2 and 3 running on TRX where I play some cutscenes with the Italian dubbing (the console is very useful, thanks!).

Tomb Raider wasn't multilingual, each CD only had one language. Maybe you thought I was asking for support for multiple dubbings at the same time? That would be nice, but I understand it would take time that could perhaps be spent elsewhere and, as already mentioned, dubbing in other languages ​​already works.

What I was talking about only concerns Tomb Raider 3 (not Tomb Raider 2, I don't know about Tomb Raider 1 because there's no Italian version) in which the FMVs have different suffixes depending on the language. I've noticed that TRX currently only recognizes videos with the `_Eng` suffix. By renaming the respective Italian files with that suffix, TRX plays them normally. It would be sufficient for TRX to detect TR3 video files with other suffixes (`_Ita` for Italian, I don't know which ones Spanish, French, and German use).
The files in question are:

- `Intr_Eng.rpl` (`.avi` or another extension if taken from the PS1 version)
- `Sail_Eng.rpl`
- `Crsh_Eng.rpl`

These are the same for all languages:

- `Logo.rpl`
- `Endgame.rpl`

EDIT
Now I understand the difficulty (it was enough to say it, I'm not that stupid 😂)... the FMVs are specified in `gameflow.json5`:
```json
    "fmvs": [
        {"path": "fmv/logo.rpl", "legal": true},
        {"path": "fmv/intr_eng.rpl"},
        {"path": "fmv/sail_eng.rpl"},
        {"path": "fmv/crsh_eng.rpl"},
        {"path": "fmv/endgame.rpl"},
    ],
```
I thought it was hardcoded in the source code. In this case, we can change the names manually until a more practical solution is found.

**Comment by rr-** [comment]
I already told you it's more complex than that. 

One example: we'll need subtitles.
Subtitles will need different timing per different UI languages vs audio languages because they don't align up.

Other example: yes deciding the suffix depending on the audio language.
Third example: discovering what audio languages the player has installed to begin with.
Fourth example: how to choose the music track file (I get that you "only" want Italian, but I want to play Japanese with English subs. This needs to be sorted out conceptually BEFORE we start implementing things).

Locking this as well – and please stop bringing this up – it's not helping.
